### PR TITLE
Review pull request

### PR DESCRIPTION
**Pull request by mprotic123**
This pull request is created in order to review the project from Microverse called "Personal Portfolio - desktop version".

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -396,20 +397,23 @@ button:hover {
...
 }
 
 .card_img3 {
   width: 370px;
   height: 200px;
-  background-image: url("../img/icon-languages.svg");
+  background-image: url("../img/skills-icon.svg");
   background-repeat: no-repeat;
+  margin: auto;
 }
```
Comment (carlos-ssh) [inline styles/style.css:418]: - Kindly use your media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 992px - 1100px). It's required in your project.

- Be DRY. your CSS file it's too long. If you added there your reset.css kindly create a new css files dedicated to this and reuse your code.

In your mobile view kindly fix this issue: 
Screen Shot 2021-07-22 at 03 57 34

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mprotic123 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -347,7 +350,7 @@ <h1 class="exp_main">
           <a href="#"><img src="icons/angellist.svg" alt="angellist logo" /></a>
         </li>
       </ul>
-      <p class="ma_mail">myemailhere@email.com</p>
-    </section>
+      <p class="ma_mail">My e-mail: milan.protic@ts9maj.edu.rs</p>
```
Comment (carlos-ssh) [inline index.html]: - You can use <a href="mailto:your@mail.com"></a> instead a p-tag.

File: index.html [hunk 3]
```diff
@@ -329,7 +332,7 @@ <h1 class="exp_main">
       <br>
     </section>
     <hr class="divide">
-    <section>
+    <div class="socials">
```
Comment (carlos-ssh) [inline index.html:335]: **- You can try to fix this using display flex.**

Screen Shot 2021-07-22 at 05 00 57

File: styles/reset.css [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+        margin: auto;
+      }
+      
+      .card_img3 {
+        width: 370px;
+        height: 200px;
+        background-image: url("../img/skills-icon.svg");
+        background-repeat: no-repeat;
+        margin: auto;
+      }
```
Comment (carlos-ssh) [inline styles/reset.css:121]: [OPTIONAL]
- You can **DRY** your code in examples like these. Where you are repeating many lines  of code.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mprotic123, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**